kvm-simple-init

```diff
@@ -185,6 +185,8 @@ action="$1"
 retcode=0
 
 for vmname in $vmnames; do
+	vmname=${vmname%.conf}
```

Comment:
You're right !
But then the vmnames content is inconsistent whether the script is called without a VM name (so vmnames contains names with .conf), or with a vmname (so vmnames contains names without .conf).
So i would suggest replacing the line with xargs by something like :
    
    vmnames=$(find ${CONFDIR} -type f -name "*.conf" \
    | while read conffile; do basename $conffile .conf; done)

And removing the line :

    vmname=${vmname%.conf}

This also removes an error if no config file are present.